### ISSUE TITLE
[WebToolsE2E][Aspire] The content of "Actions" is missing when “dotnet watch” the Aspire starter project

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#6286]
**INSTALL STEPS**

1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 9.0.100-rc.2.24474.11
3. Install Aspire 9.0.0-rc.1.24511.1

    - Apply the dotnet8 feed for Aspire 9.0

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp
```
2. Dotnet watch the project in CMD.
`dotnet watch`
3. Click the apiservice/webfrontend project "Actions"

**ACTUAL**
The content of "Actions" is missing.
![Image](https://github.com/user-attachments/assets/e8d0ee76-6a03-461e-8584-023ac7843715)

**EXPECTED**
The "Action" should display six items as below
![Image](https://github.com/user-attachments/assets/7daead97-9f42-46c1-93e1-e1673587f558)

**Comment by davidfowl** [comment]
AFAIK those actions show up when telemetry shows up for those resources. Did you make requests already?

cc @JamesNK

**Comment by joperezr** [comment]
Discussed this during our weekly meeting and mentioned that one option could be to have those options grayed out when data isn't available yet.

**Comment by Rita003** [comment]
This issue fixed on the latest Aspire 9.0.0-rc.2.24517.3.
